templates/partials/search-results-candidate.html

```diff
@@ -26,7 +26,7 @@ <h3 class="result__title">
     <div class="result__column">
       {% if candidate.principal_committees %}
         {% with committee=candidate.principal_committees[0] %}
-        <span class="t-block t-bold">Principal Campaign Committee:</span>
+        <span class="t-block t-bold">Principal campaign committee:</span>
```

Comment:
@noahmanger, @amykort and I have no preference and defer to @emileighoutlaw and whatever she thinks users will find most approachable/least off-putting.  I believe Emily thought we should capitalize as little as possible.